metabase/metabase
# setup rendering pipeline for echarts static waterfall

### Description

The first PR for the echarts waterfall chart, and a prerequisite step for https://github.com/metabase/metabase/issues/35810.  This PR just sets up the rendering pipeline, so sending a subscription email with the waterfall chart will use the new codepath, which just renders a placeholder that looks like a bar chart for now.

### How to verify

1. Create a waterfall chart
2. Add the question to a dashboard
3. Send a subscription email
4. Confirm you see the placeholder

### Demo

![Screenshot 2023-12-17 at 1.28.28 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/b8c6e7d4-5de4-45fc-8008-a8bb0118209d.png)


### Checklist


Added a storybook story. Right now it just shows the placeholder, after the next PR it will start to render the actual waterfall series.
